install.sh

```diff
@@ -78,20 +92,21 @@ https://hosts-file.net/ad_servers.txt
 https://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&showintro=0&mimetype=plaintext
 EOF
                 echo "[i] File created."
+                chmod u=rwx,g=rw,o=rw "hostssources.lst" # Allows the user to edit the file which is created by root.
```

Comment:
Thanks for using this (imho better human readable) syntax for `chmod`. But I think `hostssources.lst` musn't be edited by users that are not root. Think of a multi user system or internet cafe. If everybody is allowed to change this file, everybody has the power to block hosts for all users.